core/domain/user_services.py

```diff
@@ -359,7 +359,7 @@ def get_users_settings(user_ids):
             result.append(UserSettings(
                 feconf.SYSTEM_COMMITTER_ID,
                 email=feconf.SYSTEM_EMAIL_ADDRESS,
-                role=feconf.ROLE_ID_ADMIN,
+                role=feconf.ROLE_ID_EXPLORATION_EDITOR,
```

Comment:
in email_manager_test.EmailRightsTest.test_sender_id_validation

It expects an explicit check for system committer (which is done in validations in email_manager). It doesn't expect SYSTEM_COMMITTER to do admin or moderator tasks.